### ISSUE TITLE
Ligatures for Kotlin nested generics with wildcards

### ISSUE THREAD

**Issue by Anton3** [tonsky/FiraCode#690]
Let's look at an example with generics and wildcards in Kotlin:

```kotlin
fun collectResponses(): List<*>
```

Symbols `<*>` form a nice ligature. But let's add nested generics:

```kotlin
fun collectResponse(): List<Option<*>>
```

Currently, that last bit is displayed as `<*` `>>`, which is worse than any typical non-ligature font.

It should be displayed like `<*>` `>` instead.

**Comment by tonsky** [comment]
Agree. I’ll take a look. Thanks!

**Comment by tonsky** [comment]
I’m pretty sure it was fixed in #410. Please make sure you are using the latest version of the font.
<img width="382" alt="Screenshot 2019-09-02 at 16 33 36" src="https://user-images.githubusercontent.com/285292/64118040-6df86400-cd9f-11e9-9e2e-78df384fb749.png">
